raspberry/checkout/src/checkout_logic.py

```diff
@@ -15,14 +18,43 @@ def add_scanned_tag(self, tag) -> bool:
         return True
     
     def remove_last_scanned(self):
```

Comment:
do not change the logic of removing the last scanned. although it should be named better. this is for removing the last scanned TAG, when the server responded with the status NOK. so there is no product to remove, nor the price, cuz the tag was not connected to the product. so ONLY the tag needs to be removed